e2e/envoy-config.yaml

```diff
@@ -31,7 +31,7 @@ static_resources:
                           "@type": "type.googleapis.com/google.protobuf.StringValue"
                           value: |
                             {
-                              "rules":"SecDebugLogLevel 5 \nSecDebugLog modsec.log \nSecRuleEngine On \nSecRule REQUEST_URI \"@streq /admin\" \"id:101,phase:1,t:lowercase,deny\""
```

Comment:
https://github.com/corazawaf/coraza/issues/330

(`log.Logger` doesn't seem to work in wasm reliably, sigh - but we should use the envoy functions anyways since they're more efficient)